web/packages/plugins/analysis-platform/src/modules/Analysis/components/AnalysisJobTable.tsx

```diff
@@ -132,11 +130,7 @@ const JobRow = (props: {
   return (
     <Table.Row>
       <Table.Cell onClick={viewJob}>
-        {jobStatus !== EJobStatus.CREATED
-          ? job.started
-            ? new Date(job.started).toLocaleString(navigator.language)
-            : started
-          : 'Not started'}
+        {jobStatus !== EJobStatus.CREATED ? job.started : 'Not started'}
```

Comment:
Should this code be added? `new Date(job.started).toLocaleString(navigator.language)